var/spack/repos/builtin/packages/intel-mkl/package.py

```diff
@@ -69,15 +78,27 @@ def blas_libs(self):
 
         mkl_threading = ['libmkl_sequential']
 
+        omp_libs = LibraryList()
```

Comment:
looks like it does not have a default constructor, Python says: `Error: TypeError: __init__() takes exactly 2 arguments (1 given)`.